### PR TITLE
Started release notes for 2.1.2

### PR DESCRIPTION
**Pull request by dlmarion**

### CODE REVIEW

**Review by DomGarguilo — APPROVED** [review]
Looks good overall. None of these suggestions are critical and I think this can be merged in as is.

The only other thing I was was some of the bullet points have a period at the end and some do not. It would probably be good to make that consistent one way or another.

File: _posts/release/2023-08-14-accumulo-2.1.2.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+MANAGER_TABLET_GROUP_WATCHER_INTERVAL, to make this wait time configurable.
+
+Improvements that help with administration:
+
+* {% ghi 3678 %}, {% ghi 3683 %} Added validation of property table.class.loader.context when it is set.
+If the context name is null, an empty string, results in an exception creating the classloader for the
+context, or if the resulting classloader is null, then an exception will be returned and the property
+will not be set.
+
+* {% ghi 3548 %}, {% ghi 3561 %} Displayed Manager state prominently on the Monitor UI.
```
[the remark this answers is not in the document]
Comment (dlmarion) [reply _posts/release/2023-08-14-accumulo-2.1.2.md:41]: Right, so I wasn't sure which pages the banner was on. If you want to craft up some language I can add it in.

File: _posts/release/2023-08-14-accumulo-2.1.2.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: Apache Accumulo 2.1.2
+sortableversion: '02.01.02'
+LTM: true
```
Comment (ctubbsii) [inline _posts/release/2023-08-14-accumulo-2.1.2.md:4]:
```suggestion
LTM: true
draft: true
```

**Review by ctubbsii — CHANGES_REQUESTED** [review]
This can be merged with the draft tag in the front matter. After that, we can polish it up as needed.

**Comment by ctubbsii** [comment]
Merged as draft so it can be viewed on the site, and polished from there.

**Pull request merged**